x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx

```diff
@@ -145,6 +146,8 @@ const FlyoutHeaderPanelComponent: React.FC<FlyoutHeaderPanelProps> = ({ timeline
 
   const { euiTheme } = useEuiTheme();
 
+  console.log({ isDataInTimeline, test: combinedQueries?.filterQuery });
```

Comment:
nit: consider removing this if it's no longer required

Reply:
sorted.